### PR TITLE
Added test for rename and set properties TestFakerQueries.java

### PR DESCRIPTION
**Pull request by kavyabala23**
## Release notes

(x) This is not user-visible or is docs only, and no release notes are required.

### CODE REVIEW

File: plugin/trino-faker/src/test/java/io/trino/plugin/faker/TestFakerQueries.java [hunk 1]
```diff
@@ -35,6 +35,30 @@ void testShowTables()
         assertUpdate("CREATE TABLE faker.default.test (id INTEGER, name VARCHAR)");
         assertTableColumnNames("faker.default.test", "id", "name");
     }
+    @Test
```
Comment (ebyhr) [inline plugin/trino-faker/src/test/java/io/trino/plugin/faker/TestFakerQueries.java:38]: Please fix code styles: https://github.com/trinodb/trino/blob/master/.github/DEVELOPMENT.md#code-style

File: plugin/trino-faker/src/test/java/io/trino/plugin/faker/TestFakerQueries.java [hunk 2]
```diff
@@ -35,6 +35,30 @@ void testShowTables()
...
+       assertQuery("SHOW TABLES FROM faker.default LIKE 'original_table'", "SELECT '' WHERE false");
+       assertUpdate("DROP TABLE faker.default.renamed_table");
+    }
+    @Test
+    void testAlterTableSetProperties() 
+    {
+        assertQuery("SHOW TABLES FROM faker.default LIKE 'test_table'", "SELECT '' WHERE false");
+        assertUpdate("CREATE TABLE faker.default.test_table (id INTEGER, name VARCHAR)");
+        assertQuery("SHOW TABLES FROM faker.default", "VALUES 'test_table'");
+        assertUpdate("ALTER TABLE faker.default.test_table SET PROPERTIES comment = 'This is a test table'");
```
Comment (ebyhr) [inline plugin/trino-faker/src/test/java/io/trino/plugin/faker/TestFakerQueries.java:55]: The connector doesn't support setting `comment` table property. It should be `null_probability` or `default_limit`.

File: plugin/trino-faker/src/test/java/io/trino/plugin/faker/TestFakerQueries.java [hunk 3]
```diff
@@ -35,6 +35,29 @@ void testShowTables()
...
+        assertUpdate("DROP TABLE faker.default.renamed_table");
+    }
+    
+    @Test
+    void testAlterTableSetProperties() 
+    {
+        assertQuery("SHOW TABLES FROM faker.default LIKE 'test_table'", "SELECT '' WHERE false");
+        assertUpdate("CREATE TABLE faker.default.test_table (id INTEGER, name VARCHAR)");
+        assertUpdate("ALTER TABLE faker.default.test_table SET PROPERTIES default_limit = 100");
+        assertQuery("SELECT table_property_name, table_property_value FROM information_schema.tables WHERE table_name = 'test_table' AND table_property_name = 'default_limit'",
```
Comment (nineinchnick) [inline plugin/trino-faker/src/test/java/io/trino/plugin/faker/TestFakerQueries.java:57]: The table_property_name column doesn't exist. I think you have to execute a show create table query instead

File: plugin/trino-faker/src/test/java/io/trino/plugin/faker/TestFakerQueries.java [hunk 4]
```diff
@@ -35,6 +35,28 @@ void testShowTables()
         assertUpdate("CREATE TABLE faker.default.test (id INTEGER, name VARCHAR)");
         assertTableColumnNames("faker.default.test", "id", "name");
     }
+    
+    @Test
+    void testRenameTable()
+    {
+        assertQuery("SHOW TABLES FROM faker.default LIKE 'original_table'", "SELECT '' WHERE false");
```
Comment (nineinchnick) [inline plugin/trino-faker/src/test/java/io/trino/plugin/faker/TestFakerQueries.java:42]: This is not formatted correctly, you need to add one more space for indentation. Try setting up your IDE to format all files automatically.

**Comment by nineinchnick** [comment]
@kavyabala23 can you make sure tests pass, before pushing more updates to this branch? If you're having issues with running tests locally, ask for help on the Trino Slack.

**Comment by ebyhr** [comment]
@cla-bot check

**Comment by nineinchnick** [comment]
@kavyabala23 please take a look at the failures in the CI workflow: 
```
Error:  src/test/java/io/trino/plugin/faker/TestFakerQueries.java:[21,15] (imports) UnusedImports: Unused import - org.assertj.core.api.Assertions.assertThat.
Error:  src/test/java/io/trino/plugin/faker/TestFakerQueries.java:[55,164] (whitespace) NoWhitespaceBefore: ',' is preceded with whitespace.
```

**Pull request closed without merging**